[2022-07-21 06:30:24,156] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: etl_dag.load manual__2022-07-21T06:30:12.867044+00:00 [queued]>
[2022-07-21 06:30:24,170] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: etl_dag.load manual__2022-07-21T06:30:12.867044+00:00 [queued]>
[2022-07-21 06:30:24,171] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-21 06:30:24,171] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-07-21 06:30:24,171] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-21 06:30:24,190] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): load> on 2022-07-21 06:30:12.867044+00:00
[2022-07-21 06:30:24,196] {standard_task_runner.py:52} INFO - Started process 213 to run task
[2022-07-21 06:30:24,200] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl_dag', 'load', 'manual__2022-07-21T06:30:12.867044+00:00', '--job-id', '73', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpshz4misv', '--error-file', '/tmp/tmps9jrk_a5']
[2022-07-21 06:30:24,202] {standard_task_runner.py:80} INFO - Job 73: Subtask load
[2022-07-21 06:30:24,309] {task_command.py:371} INFO - Running <TaskInstance: etl_dag.load manual__2022-07-21T06:30:12.867044+00:00 [running]> on host 4514cd19035a
[2022-07-21 06:30:24,431] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_dag
AIRFLOW_CTX_TASK_ID=load
AIRFLOW_CTX_EXECUTION_DATE=2022-07-21T06:30:12.867044+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-21T06:30:12.867044+00:00
[2022-07-21 06:30:28,348] {python.py:173} INFO - Done. Returned value was: None
[2022-07-21 06:30:28,367] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=etl_dag, task_id=load, execution_date=20220721T063012, start_date=20220721T063024, end_date=20220721T063028
[2022-07-21 06:30:28,423] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-21 06:30:28,471] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
